.npmrc

```diff
@@ -1,2 +1,3 @@
 engine-strict=true
 resolution-mode=highest
+auto-install-peers=true
```

Comment:
What does this do?

Reply:
Just a random fixup. One of those _since I was there's_.